controller/src/main/java/org/jboss/as/controller/audit/ManagedAuditLogger.java

```diff
@@ -182,6 +184,14 @@
     JsonAuditLogItemFormatter getJsonFormatter(String name);
 
     /**
+     * Gets the last log entries
+     *
+     * @param name the name of the handler
+     * @return the last log entries of the handler
+     */
+    List<ModelNode> listLastEntries(String name);
```

Comment:
Why not use a default method here?
```java
default List<ModelNode> listLastEntries(String name) {
    return Collections.emptyList();
}
```

This would allow you not to have to define `Collections.emptyList()` everywhere else.